### ISSUE TITLE
How long time do you spend to train the model? 

### ISSUE THREAD

**Issue by Pengyang233** [zhangboshen/DenseNet-human-pose-estimation#2]
Hi boshen:
   i fork your project and run on my own datasets. i resize the train images to [200,200],  but the number of  feature points (label) in my trainsets is 16 points. I modified your code in order to meet my demand: all 22 in your code are changed to 32. I have two questions to be solved.
1. How long time do you spend to train the model? I run on two 1080Ti GPUs(10G memory each),i have trained 50000 steps, but the loss is still10000, which I think is a little high. I want to know the time or steps  you spent on training.
2. The label. I see your label is XXX.mat with [num_img, num_points, 2].
    Assume that my label file is Label.mat. so the size(Label) = [6131,16,2]
    and the Label[ : , : , 1] represents the x-coordinate of the featue points ( row in image).
                 Label[ : , : , 2] represents the y-coordinate of the featue points ( col in image ).
    Is the format of my Label the same as yours? 

  Thank you very much.

**Comment by zhangboshen** [comment]
Well, like I said, this code is kindof untrimmed, 50K training step, which will train my dataset 10 epochs, and yes, it converged when I trained my model, but I think learning rate is pretty crucial if u want a fast convergence, so u should try some more different learning rate.  
Second, yes, ur label is the same as mine, it is a pretty common label format, no other tricks here.
And, good luck.

**Comment by zhangboshen** [comment]
@Pengyang233
